### PR TITLE
Dashboard Sidebar and Animation

### PR DESCRIPTION
**Pull request by konradscheng**
https://app.asana.com/0/1170551505628299/1189828027572753/f

Currently waiting on icons to replace button for opening/closing sidebar.

### CODE REVIEW

File: src/components/dashboard/sidebar/sidebar-layout.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+`;
+
+let PageContent = styled.div`
+  max-width: 95%;
+  width: ${ props => props.sidebarOpen ? "87%" : "95%"};
+  transition: width 0.5s ease;
+`;
+
+let Sidebar = styled.div`
+  width: ${ props => props.sidebarOpen ? "13%" : "5%"};
```
Comment (Tony1023) [inline src/components/dashboard/sidebar/sidebar-layout.js:25]: Would you think that it's better & easy to change pixel values for the sidebar width?

File: src/components/grid.js [hunk 2]
```diff
@@ -12,11 +12,12 @@ import styled from "styled-components"
    flex-basis: ${({width}) => width ? width : 'auto'};
    flex-grow: ${({width}) => width ? 0 : 1};
    box-sizing: border-box;
+   max-width: 88%;
```
Comment (Tony1023) [inline src/components/grid.js:15]: Ditto

File: src/components/dashboard/sidebar/sidebar-layout.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  box-shadow:         1px 2px 4px 5px #ccc;
+`;
+
+let SidebarItem = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 10px;
+  margin-top: 5%;
+  margin-bottom: 5%;
```
Comment (Tony1023) [inline src/components/dashboard/sidebar/sidebar-layout.js:41]: Let's add cursor: pointer ?

File: src/pages/dashboard/restaurant.js [hunk 4]
```diff
@@ -29,36 +29,16 @@ let styles = {
 }
 
 const Restaurant = () => (
-  <Layout>
+  <SidebarLayout>
```
Comment (Tony1023) [inline src/pages/dashboard/restaurant.js:32]: Are we missing an entry point for this page?

File: src/pages/dashboard/personal.js [hunk 5]
```diff
@@ -28,36 +28,16 @@ let styles = {
 }
 
 const Personal = () => (
-  <Layout>
+  <SidebarLayout>
```
Comment (Tony1023) [inline src/pages/dashboard/personal.js:31]: This page is mysteriously vertically scrollable...

**Review by Tony1023 — CHANGES_REQUESTED** [review]
Thanks for the PR!

Side bar and its expand/collapse animations looks great on my computer, but there are still a few things we might want to take a look. I left some comments below :)

**Review by cchyung — APPROVED** [review]

**Pull request merged**